src/eventstream-proxy/eventstream-proxy.interfaces.ts

```diff
@@ -26,7 +26,7 @@ export interface ConnectionListener {
 }
 
 export interface EventListener {
-  onEvent: (subName: string, event: Event, process: EventProcessor) => void | Promise<void>;
+  onEvent: (subName: string, event: Event) => undefined | Promise<WebSocketMessage | undefined>;
```

Comment:
FYI in ERC1155 we have some event handlers that might actually return _multiple_ messages to be passed to FireFly in response to a single blockchain event (because ERC1155 supports batched transfers). That means when this is ported to ERC1155, I think we'll have to return `WebSocketMessage[]`.

That doesn't necessarily mean you need to do so here, but wanted to point it out for potential consistency.